_vandv/swbli.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Shock-Wave Boundary-Layer Interaction
+permalink: /vandv/swbli/
+---
+
+| Solver | Version | Author |
+| --- | --- | --- |
+| `RANS` | 7.4.0 | P. Gomes |
+
+The details of the Mach 5 SWBLI validation case are taken from [NASA](https://www.grc.nasa.gov/www/wind/valid/m5swbli/m5swbli.html).
+
+<p align="center">
+<img src="/vandv_files/swbli/mach.png" alt="Mach number contours (SST-2003m)" />
+</p>
+**Figure 1** - Mach number contours (SST-2003m).
+
+We validate our implementation of the SA and SST models by comparing the SU2 numerical results on a sequence of grids against experimental results.
+
+## Problem Setup
+
+The geometry and flow conditions are according to the [main reference](https://www.grc.nasa.gov/www/wind/valid/m5swbli/m5swbli.html).
+The SU2 configuration files used in this study are [available here](https://github.com/su2code/SU2/blob/develop/TestCases/vandv/rans/swbli/).
+Mean flow convective fluxes were computed with Roe's scheme and a limited MUSCL reconstruction (Green-Gauss gradients and Venkatakrishnan's limiter).
```

Comment:
I guess you want to adapt this line: Roe or JST and Venkatakrishnan or van Aldaba limter